README.md

```diff
@@ -23,10 +23,27 @@
 1. Roadmap
 
 ## What is BigFlow?
-TODO
+
+BigFlow is a Python framework built on top of the [Cloud Composer](https://cloud.google.com/composer) ([Airflow](https://airflow.apache.org/)).
+It is made to simplify developing data processing pipelines on GCP.
+
+The main features are:
+
+* [Dockerized jobs](./docs/project_setup_and_build.md#overview)
+* [Powerful CLI](./docs/cheatsheet.md)
+* [Automated build](./docs/project_setup_and_build.md#overview), [deployment](./docs/deployment.md), 
```

Comment:
, [configuration] -> , **and** [configuration]

Reply:
done